pkg/cluster/cluster.go

```diff
@@ -43,6 +45,7 @@ type Peer struct {
 const (
 	DefaultPushPullInterval = 5 * time.Second
 	DefaultGossipInterval   = 5 * time.Second
+	DefaultRefreshInterval  = 30 * time.Second
```

Comment:
maybe minute for it? or something bit longer? Not sure.

Reply:
Yeah, a minute should be fine. We are acquiring a lock (https://github.com/improbable-eng/thanos/pull/383/files#diff-1a365fc2cc3058f4efcd7c1457c216c2R239), so less frequent runs make sense. And it shouldn't increase startup time as it take awhile for every component to join the cluster.